test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -282,6 +282,13 @@ name: lua
       else
         request_handle:headers():add("request_secure", "true")
       end
+      local ok, error = request_handle:streamInfo():setSni("foo.bar")
```

Comment:
Could we add more realistic test? It means that 1. extract header value (it should be hostname for upstream SNI) 2. call `setSni`

Reply:
Sure, let me do that.